test/StressTestMetrics/README.md

```diff
@@ -0,0 +1,12 @@
+# OpenTelemetry Metrics Stress Run
+
+Use the following example to run Stress test from command line:
+
+Navigate to `./test/StressTestMetrics` directory and run the following command:
+
+`dotnet run --framework netcoreapp3.1 --configuration Release`
+
+The program shows the writes/sec on the console window title
```

Comment:
One suggestion on the behavior:
1. the application will keep running until you terminate it.
2. the application will show both the current writes/sec (based on 1 sec internal) and the avg writes/sec (since the beginning) on the console window title.

Reply:
Yes I'll make this in future PRs.